### DISCUSSION TITLE
Can the Dutch Eredivies (Soccer) be added

### DISCUSSION THREAD

**Discussion by ml88nl** [vasqued2/ha-teamtracker#61] [General]
Hi All,

Can the Dutch Eredivisie(ned.1) be added as a default selectable League?

**Comment by vasqued2** [comment]
I’m not adding native support for more soccer  leagues.  It’s a never ending cycle for everyone’s favorite.  

Just add it as a Custom API Configuration.  That’s what it’s there for.  There’s no difference in functionality once it’s set up anyways.

Sorry.
.
